src/packages/core/src/connector-loader.ts

```diff
@@ -17,9 +17,9 @@ const initialize = <T extends FlavorName = typeof DefaultFlavor>(
   let asyncRequestProcessing: boolean;
 
   if ("chain" in options && "asyncRequestProcessing" in options["chain"]) {
-    asyncRequestProcessing = options["chain"].asyncRequestProcessing;
+    asyncRequestProcessing = options.chain.asyncRequestProcessing;
```

Comment:
Once upon a time typescript couldn't figure out the types here, now it can. This is a weird change to have in this PR and I can remove it if anyone objects.